chapters/snippet-information.md

```diff
@@ -229,19 +229,14 @@ Supported classes from the pointer method vocabulary are `StartEndPointer` and `
 
 This field contains the license the SPDX document creator has concluded as governing the snippet or alternative values if the governing license cannot be determined. The options to populate this field are limited to:
 
-A valid SPDX License Expression as defined in Annex [D](SPDX-license-expressions.md).
+* A valid SPDX License Expression as defined in Annex [D](SPDX-license-expressions.md).
```

Comment:
```suggestion
* A valid SPDX License Expression as defined in [Annex D](SPDX-license-expressions.md).
```

Reply:
Not sure this should be blocking the merge change.